src/python/pants/notes/1.13.x.rst

```diff
@@ -2,6 +2,11 @@
 ======================
 This document describes releases leading up to the ``1.13.x`` ``stable`` series.
 
+1.13.0 (12/21/2018)
```

Comment:
I guess that our guidance on https://www.pantsbuild.org/release_strategy.html#stable-release-candidates isn't really clear as to whether the 5 days are for "all of the rcs" or for "each of the rcs". I guess I'm fine with the former, but having some minimum time for "each" rc might be good too.

Reply:
Yeah, I was operating under the assumption of "The last RC was a trivial cherry-pick which mostly just exposed a new option", so not worrying to much about it.